### PR TITLE
commits changes made to leverage bulk_ methods for package_load, in p…

### PR DESCRIPTION
**Pull request by whatisgalen**
…articular skos and concept-related operations, re #5487

### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

speeds up some operations called by package_load

### Issues Solved

#5487 

### Checklist

-   [x] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: 
*   Found by: @ 
*   Tested by: @ 
*   Designed by: @

### CODE REVIEW

**Comment by whatisgalen** [comment]
Note that the stickler annotation "line break before binary operator" arises because the linter is set to pep8; the files changed in this PR were formatted with black.

File: arches/app/utils/skos.py [hunk 1]
```diff
@@ -120,198 +184,429 @@ def save_concepts_from_skos(self, graph, overwrite_options='overwrite', staging_
...
+                            node.save()
+                            # concepts.append(node)
+                        elif overwrite_options == "ignore":
+                            try:
+                                # don't do anything if the concept already exists
+                                models.Concept.objects.get(pk=node.conceptid)
+                            except:
+                                # else save it
+                                node.save()
+                                # concepts.append(node)
```
Comment (apeters) [inline arches/app/utils/skos.py]: this can safely be removed

File: arches/app/utils/skos.py [hunk 2]
```diff
@@ -120,198 +184,429 @@ def save_concepts_from_skos(self, graph, overwrite_options='overwrite', staging_
...
 
                 # need to index after the concepts and relations have been entered into the db
                 # so that the proper context gets indexed with the concept
                 if scheme_node:
                     scheme_node.bulk_index()
 
-
             # insert the concept collection relations
             # we do this outide a transaction so that we can load incomplete collections
+            # relation_objs = []
```
Comment (apeters) [inline arches/app/utils/skos.py]: remove unused vars

File: arches/app/utils/skos.py [hunk 3]
```diff
@@ -120,198 +184,429 @@ def save_concepts_from_skos(self, graph, overwrite_options='overwrite', staging_
...
-                        conceptto_id=relation['target'],
-                        relationtype_id=relation['type']
+                    # if bulk_load is True:
+                        # newrelation = models.Relation(
+                        #     conceptfrom_id=relation['source'],
+                        #     conceptto_id=relation['target'],
+                        #     relationtype_id=relation['type']
+                        # )
+                        # relation_objs.append(newrelation)
+                    # else:
```
Comment (apeters) [inline arches/app/utils/skos.py]: this can be removed

File: arches/app/utils/skos.py [hunk 4]
```diff
@@ -120,198 +184,429 @@ def save_concepts_from_skos(self, graph, overwrite_options='overwrite', staging_
...
+                        conceptfrom_id=relation["source"],
+                        conceptto_id=relation["target"],
+                        relationtype_id=relation["type"],
                     )
                 except IntegrityError as e:
                     self.logger.warning(e)
+                    pass
 
+            # if bulk_load is True:
+            #   models.Relation.objects.bulk_create(relation_objs, ignore_conflicts=True)
```
Comment (apeters) [inline arches/app/utils/skos.py]: remove this too

**Pull request merged**